pt_miniscreen/pages/network/ap.py

```diff
@@ -9,16 +9,16 @@ class Page(PageBase):
     def __init__(self, size):
         row_data = NetworkPageData(
             first_row=RowDataText(
-                icon_path="sys_info/networking/antenna.png",
-                text=lambda: get_ap_mode_status().get("ssid", ""),
+                icon_path="sys_info/networking/antenna-small.png",
```

Comment:
i think this should be the wifi icon, same as we use for ssid on the wifi page

Reply:
Yeah agreed! Done :)